components/org.wso2.carbon.appmgt.mdm.osgiconnector/src/main/java/org/wso2/carbon/appmgt/mdm/osgiconnector/mdmmgt/util/AndroidApplicationOperationUtil.java

```diff
@@ -0,0 +1,112 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wso2.carbon.appmgt.mdm.osgiconnector.mdmmgt.util;
+
+import org.wso2.carbon.appmgt.mdm.osgiconnector.mdmmgt.beans.MobileApp;
+import org.wso2.carbon.appmgt.mdm.osgiconnector.mdmmgt.beans.android.AppStoreApplication;
+import org.wso2.carbon.appmgt.mdm.osgiconnector.mdmmgt.beans.android.EnterpriseApplication;
+import org.wso2.carbon.appmgt.mdm.osgiconnector.mdmmgt.beans.android.WebApplication;
+import org.wso2.carbon.appmgt.mdm.osgiconnector.mdmmgt.common.DeviceApplicationException;
+import org.wso2.carbon.device.mgt.common.operation.mgt.Operation;
+import org.wso2.carbon.device.mgt.core.operation.mgt.ProfileOperation;
+
+/**
+ * This class contains the all the operations related to Android.
+ */
+public class AndroidApplicationOperationUtil {
+
+	/**
+	 * Create Install Application operation.
+	 *
+	 * @param application MobileApp application
+	 * @return operation
+	 * @throws DeviceApplicationException
+	 */
+	public static Operation createInstallAppOperation(MobileApp application) throws
+	                                                                         DeviceApplicationException {
```

Comment:
In `createAppUninstallOperation` method we do have some identical code to this method. Create the new method and move redundant code to that method so that we can re-use the method.